docs/en/algorithm_introduction/knowledge_distillation.md

```diff
@@ -0,0 +1,94 @@
+# Knowledge Distillation
+
+---
+## Content
+
+* [1. Introduction of model compression methods](#1)
+* [2. Application of knowledge distillation](#2)
+* [3. Overview of knowledge distillation methods](#3)
+    * [3.1 Response based distillation](#3.1)
+    * [3.2 Feature based distillation](#3.2)
+    * [3.3 Relation based distillation](#3.3)
+* [4. Reference](#4)
+
+<a name='1'></a>
+
+## 1. Introduction of model compression methods
+
+In recent years, deep neural networks have been proved to be an extremely effective method for solving problems in the fields of computer vision and natural language processing. A suitable neural network architecture might performs better than traditional algorithms mostly.
+
+When the amount of data is large enough, increasing the model parameters with a reasonable method can significantly improve the model performance, but this brings about the problem of a sharp increase of the model complexity. It costs more for larger models.
+
+Parameter redundancy exists in deep neural networks generally. At present, there are several mainstream methods to compress the model and reduce parameters. Such as pruning, quantization, knowledge distillation, etc. Knowledge distillation refers to the use of a teacher model to guide the student model to learn specific tasks to ensure that the small model obtains relatively large performance, and even has comparable performance with the large model [1].
+
+
+Currently, knowledge distillation methods can be roughly divided into the following three types.
+
+* Response based distillation: Output of student model is guided by the teacher model for
+* Feature based distillation: Inner feature map of student model is guided by the teacher model.
+* Relation based distillation: For different samples, the teacher model and the student model are used to calculate the correlation of the feature map between the samples, the final goal is to make sure that correlation matrix of student model and the teacher model are as consistent as possible.
+
+
+<a name='2'></a>
+
+## 2. Application of knowledge distillation
+
+Knowledge distillation algorithm is widely used in lightweight tasks. For tasks that need to meet specific accuracy, by using the knowledge distillation method, we can achieve the required accuracy with a smaller model, thereby reducing model deployment cost.
+
+
+What's more, for the same model structure, pre-trained models obtained by knowledge distillation often performs better, and these pre-trained models can also improve performance of the downstream tasks. For example, in , a pre-trained image classification model with higher accuracy can also help other tasks obtain significant accuracy gains such as target detection, image segmentation, OCR, and video classification.
```

Comment:
For example, ~~in ,~~ a pre-trained image classification model ...

Reply:
done, thx